src/app/products/_components/ProductCard.tsx

```diff
@@ -0,0 +1,50 @@
+import Icon from '@/components/common/Icon';
+
+interface CardProps {
+  // imgUrl?: string
+  brandName: string; //브랜드명
+  productName: string; //제품명
+  price: string; //가격
```

Comment:
```suggestion
  price: number
```
가격은 숫자로 받으면 좋을 것 같아요 ~!!!
숫자로 받은 다음에 밑에서 함수 처리하면 될 것 같습니둥

Reply:
050987d0872f342583b1aeea49bc5607afd3799e